configstack/running_limit.go

```diff
@@ -7,16 +7,13 @@ import (
 	"github.com/fatih/color"
 )
 
-const waitTimeBetweenThread = 2500
-
 func initWorkers(n int) {
 	if n <= 0 {
 		panic(fmt.Errorf("the number of workers must be greater than 0 (%d)", n))
 	}
 	if burstyLimiter == nil {
 		burstyLimiter = make(chan int, n)
 		for i := 1; i <= n; i++ {
-			time.Sleep(waitTimeBetweenThread * time.Millisecond) // Start workers progressively to avoid throttling
```

Comment:
This was the reason why plan-all on 1 project would take a lot more time than just plan.

Reply:
Shouldn't this whole function be reworked and/or removed ?

Reply:
Just removed, no. It is what is limiting the number of workers that work concurrently on modules.
Reworked, yes, but it will ask for more delicate modification of the rest of the code since, from what I understood, it is kind of a base for the rest of the concurrency execution. I just fear this may end up causing more problems than it fixes (if not done properly).